build.cmd

```diff
@@ -22,49 +21,54 @@ if /I "%1" == "/no-multi-proc" set MSBuildAdditionalArguments=&&shift&& goto :Pa
 call :Usage && exit /b 1
 :DoneParsing
 
-if not exist "%DeveloperCommandPrompt%" (
-  echo In order to build this repository, you need Visual Studio "15" Preview installed.
+if not exist "%VSINSTALLDIR%" (
```

Comment:
The variable needs to be added [here](https://devdiv.visualstudio.com/DevDiv/_build/explorer?_a=variables&definitionId=1987)